management/prompt.go

```diff
@@ -16,6 +20,73 @@ type Prompt struct {
 	WebAuthnPlatformFirstFactor *bool `json:"webauthn_platform_first_factor,omitempty"`
 }
 
+// CustomPrompt to be used for Custom Prompt Partials.
```

Comment:
I don't want to bikeshed over a name, but the [API docs](https://auth0.com/docs/api/management/v2/prompts/get-partials) don't seem to use the custom prompts naming but rather `partials`, so I'm wondering if we should match that for consistency? 

The node SDK looks to be using [partials](https://github.com/auth0/node-auth0/pull/985/files#diff-42b1a344bbd2fc34f12230a57717c19eb1157069618b09126506b5ab1fd2440fR49) and [PartialsPrompt](https://github.com/auth0/node-auth0/pull/985/files#diff-bc779360d260fbc1b28befa72be7b066034440a84d6b37758ee802ef2d6dd03bR13681) (generated from the API docs)

Reply:
I like that naming convention. Changed and pushed.